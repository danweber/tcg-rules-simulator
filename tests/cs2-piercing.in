VERSION:2
#   "evolveEffect": "\uff1cSecurity Attack +1\uff1e",
FIELD2:Attacker,CS2-Anime
SECURITY1:CS1-02 CS1-02 
GAME:P2 T1 MAIN 7
DUMP:SECURITY,TRASH

ECHO: -- attack into defender and pierce through
CMD:ATTACK Attacker Player
STEP:AUTOANSWER:CAPTURE
DUMP:SECURITY,TRASH

EXIT
