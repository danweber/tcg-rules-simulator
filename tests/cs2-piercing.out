FIELD1:
FIELD2:Attacker,CS2-Animesword
SECURITY1:CS1-Papershield CS1-Papershield
SECURITY2:
TRASH1:
TRASH2:
GAME:P2 T1 MAIN 7
ECHO: -- attack into defender and pierce through
FIELD1:
FIELD2:Attacker,CS2-Animesword,REST
SECURITY1:CS1-Papershield
SECURITY2:
TRASH1:CS1-Papershield
TRASH2:
GAME:P2 T1 MAIN 7
